demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -105,12 +140,19 @@ def verify_results(
         # get expected_values for the given query result
         td_event_id = result.pop(f"{tested_dataset}.test_data_event_id")
         expected_values = None
+        timezone_delta = 0
         for e in test_data.data:
             if str(e.test_data_event_id) == td_event_id:
                 expected_values = e.expected_values
+                timezone_delta = int(e.timezone_delta or 0)
                 break
 
         if expected_values:
+            if (expected_time_value := expected_values.get("_time")) and (time_value := result.get("_time")):  # fmt: skip
```

Comment:
`# fmt: skip` why needed?

Reply:
I didn't like the style of how `Black` formatted it.
```
if (expected_time_value := expected_values.get("_time")) and (
    time_value := result.get("_time")
):
```
But. I can revert the ignore.

Reply:
@dorschw WDYT?